### PR TITLE
include: add "zephyr/" include prefix

### PR DESCRIPTION
**Pull request by fabiobaltieri**
Add relevant "zephyr/" prefixes to allow building with LEGACY_INCLUDE_PATH=n.

### CODE REVIEW

**Comment by fabiobaltieri** [comment]
@yperess can you review this? (can't really add you as reviewer for some reason, may want to check the repo settings as well)

**Comment by yperess** [comment]
This should go upstream first

**Comment by yperess** [comment]
See https://github.com/zephyrproject-rtos/chre/pull/2#issuecomment-1126276868

I haven't had time to do it

File: platform/zephyr/context.cc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 #include "chre/platform/context.h"
 
-#include <zephyr.h>
```
Comment (gmarull) [inline platform/zephyr/context.cc:19]: I'd suggest migrating all `<zephyr/zephyr.h>` to `<zephyr/kernel.h>` as well.

Comment (fabiobaltieri) [reply platform/zephyr/context.cc:19]: Sure, may as well while I'm at it.

**Comment by fabiobaltieri** [comment]
Sent out upstream: https://android-review.googlesource.com/c/2160657 https://android-review.googlesource.com/c/2160658

**Pull request closed without merging**